spec_helper.rb

```diff
@@ -31,7 +31,7 @@ def init_publish_client(conf)
   )
 end
 
-def init_dispatcher_client(conf)
+def init_web_client(conf)
```

Comment:
changed it to a more appropriate name since we use it for author and publish tests